added the light/dark and auto them toggel in the header

samcode9 opened the pull request:
Theme Toggle Support Added in this project 
Added a theme toggle button with options in the header : Light, Dark, auto. 
Used the icons for visual clarity, 

Closes #677

yuri-pro07 commented:
![{6E05563C-167D-4F5E-B89A-0D6F323A3A94}](https://github.com/user-attachments/assets/b6912020-f003-4d9f-bf7c-d1a0f636e25c)
![{C7FAA511-84B0-4BAA-868B-BACA99853CCA}](https://github.com/user-attachments/assets/754d90b3-76f2-42e2-8724-53b54ff96e84)
![{810D7750-52D3-4BFA-9098-E65CA91797A7}](https://github.com/user-attachments/assets/a85d8162-5ef0-4bf9-8b84-8ebceae3bf0f)
Hi @samcode9. The light theme is good, but as you seen the dark theme is not working well, can you fix it please ?

marcgc21 commented:
I think that in that case much more work must be needed. 
@samcode9 if you want to keep working on that and also fixing the styling on the images feel free. However as this issue is included inside the UI GSOC project I think that it will be better to let this to someone who will work on that constantly.

Att,

M.

samcode9 commented:
@marcgc21  Thank you for the update. I would like to mention that I have already addressed and fixed the styling issues related to the images. As I have also applied for this project through GSoC, I am keen to continue working on it and contribute consistently. I believe this will ensure continuity and alignment with the project's objectives.
